client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -108,7 +108,7 @@ export function* createSite( {
 		},
 		...( bearerToken && { authToken: bearerToken } ),
 	};
-	const success = yield dispatch( SITE_STORE, 'createSite', params );
+	const success: unknown = yield dispatch( SITE_STORE, 'createSite', params );
```

Comment:
This is unfortunate. Is it not a `{ body: Site }` by any chance?

Reply:
Looks like `Site` is a namespace. `Site.NewSiteBlogDetails | undefined` seems to work and match the type of https://github.com/Automattic/wp-calypso/blob/0e29046df331416844121cc19585bdd2d68dc90b/packages/data-stores/src/site/reducer.ts#L20

(Disclaimer: I'm not 100% sure how typing in `@wordpress/data*` works)